docs/src/modules/java/pages/components.adoc

```diff
@@ -0,0 +1,28 @@
+= Akka SDK Components
+
+include::ROOT:partial$include.adoc[]
+
+== Endpoints
```

Comment:
The order of the components could be debated again, but when we decided the order in the left ToC we said that we start with Entities because it is our core value. Anyone can do endpoints.